plugins/workbench-resources/src/components/Navigator.svelte

```diff
@@ -144,7 +167,20 @@
     {/if}
 
     {#if specials.length > 0}<TreeSeparator />{/if}
-
+    {#if filteredViews && filteredViews.length > 0}
+      <TreeNode label={view.string.FilteredViews}>
+        {#each filteredViews as fV}
+          <TreeItem
+            title={fV.name}
+            on:click={() => {
+              $filterStore = JSON.parse(fV.filters)
```

Comment:
Have you tested it? I think it won't work because this filters will be stored for wrong class and after that old filters for new class will be loaded from localStorage.  You should navigate and then load filters. Or FilterBar should check filters `key._class` in `saveFilters()` and `load()`. Or better both of this solutions)